core/controllers/creator_dashboard.py

```diff
@@ -287,7 +271,6 @@ def get(self):
 class ExplorationDashboardStatsHandler(base.BaseHandler):
     """Returns the most recent open feedback for an exploration."""
 
-    @base.require_fully_signed_up
     def get(self, exploration_id):
```

Comment:
Oh, hm, I don't know. Might be worth doing a git blame to see when it got added, that might give a clue where to look. (Sometimes the URL is assembled from parts.)